src/app.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
+import 'normalize.css/normalize.css';
+
 import AppRouter from './router/AppRouter';
 import configureStore from './store/configureStore';
 import { addActivity } from './store/activities/actions';
-import 'normalize.css/normalize.css';
+import { monday, tuesday } from './constants/dates';
```

Comment:
it is a good practice to call such an importants constants with big letter `MONDAY`